src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -345,14 +353,15 @@ function ReportActionItemMessageEdit(props) {
      * @param {Event} e
      */
     const triggerSaveOrCancel = useCallback(
-        (e) => {
+        (e: NativeSyntheticEvent<TextInputKeyPressEventData>) => {
             if (!e || ComposerUtils.canSkipTriggerHotkeys(isSmallScreenWidth, isKeyboardShown)) {
                 return;
             }
-            if (e.key === CONST.KEYBOARD_SHORTCUTS.ENTER.shortcutKey && !e.shiftKey) {
+            const keyEvent = e as unknown as KeyboardEvent;
```

Comment:
Instead of this
```ts
            const keyEvent = e as unknown as KeyboardEvent;
```

Change the argument:
```ts
(e: NativeSyntheticEvent<TextInputKeyPressEventData> | KeyboardEvent)